fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -261,11 +261,14 @@ protected Response exceptionResponse(FHIROperationException e, Status status) {
             status = issueListToStatus(e.getIssues());
         }
 
+        // Only log full stack trace if server (5xx) error, or if logging level is at least FINE
         if (status.getFamily() == Status.Family.SERVER_ERROR) {
             log.log(Level.SEVERE, e.getMessage(), e);
         } else {
-            if (log.isLoggable(Level.INFO)) {
-                log.log(Level.INFO, e.getMessage(), e);
+            if (log.isLoggable(Level.FINE)) {
+                log.log(Level.FINE, e.getMessage(), e);
+            } else if (log.isLoggable(Level.INFO)) {
```

Comment:
if you flip them, it'll short circuit in an operating environment, otherwise it needs 2 boolean checks most of the time.

Reply:
also possible to push up to an else if

Reply:
Changed to else if.  Kept FINE before INFO, so FINE branch will run if logging level is fine or higher; otherwise INFO branch will run instead.

Reply:
makes sense